File: 70.climbing-stairs.py
class Solution:
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        cur = 1
        last = 1
        for _ in range(n - 1):
            tmp = cur + last
            last = cur
            cur = tmp
        return cur

File: 70.climbing-stairs.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    let cur = 1, last = 1;
    while (--n) {
        let tmp = cur + last;
        last = cur;
        cur = tmp;
    }
    return cur;
};